client/blocks/site-address-changer/index.jsx

```diff
@@ -313,42 +313,40 @@ export class SiteAddressChanger extends Component {
 						eventProperties={ { blog_id: siteId } }
 					/>
 					<Card className="site-address-changer__content">
-						<FormSectionHeading>{ translate( 'Change Site Address' ) }</FormSectionHeading>
+						<div className="site-address-changer__info">
+							<p>
+								{ translate(
+									'Once you change your site address, %(currentDomainName)s will no longer be available.',
+									{
+										args: { currentDomainName },
+									}
+								) }
+							</p>
+						</div>
+						<FormLabel htmlFor="site-address-changer__text-input">
+							{ translate( 'Enter your new site address' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **35** times:
`translate( 'enter your new site address', { context: 'logged-out-homepage'} )` *ES Score: 8*